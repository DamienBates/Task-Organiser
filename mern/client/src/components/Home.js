import React from 'react'
import { Box, Typography } from '@material-ui/core'

export const Home = () => {
    return (
        <Box ml={28} mt={-3.5} mr={1}>
            <Typography paragraph>
                Hello, you are on the homepage
            </Typography >
        </Box >
    )
}

export default Home