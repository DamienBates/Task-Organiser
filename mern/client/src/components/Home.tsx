import { Box, Typography } from '@material-ui/core'

export const Home = () => {
    return (
        <Box>
            <Typography paragraph>
                Create your task using the "Create Task" button at the top right. Indicate urgency
                and it'll sort with highest priority being at the top.
            </Typography>
        </Box>
    )
}

export default Home