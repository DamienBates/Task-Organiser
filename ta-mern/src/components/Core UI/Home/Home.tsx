import Typography from "@mui/material/Typography";
import React from "react";
import Box from "@mui/system/Box";

export default function Home() {
    return (
        <Box
            style={{
                display: "flex",
                flexDirection: "column",
                alignItems: "center",
                justifyContent: "center",
                height: "94vh",
                paddingRight: "1rem",
                paddingLeft: "1rem"
            }}>
            <Typography>
                Feel free to add, organise, edit and sort tasks to your liking.
            </Typography>
        </Box>
    )
}