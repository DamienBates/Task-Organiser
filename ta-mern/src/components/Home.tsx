import { Create, HomeOutlined } from "@mui/icons-material"
import { Typography } from "@mui/material"
import { Box } from "@mui/system"

export default function Home() {
    return (
        <Box style={{
            display: "flex",
            flexDirection: "column",
            justifyContent: "center",
            height: "94vh"
        }}>
            <Typography style={{ paddingBottom: '45px' }}>
                Welcome to my task organiser, I hope you enjoy your stay.
            </Typography>
            <Typography>
                Feel free to add, organise, edit and sort tasks to your liking.
            </Typography>
        </Box>
    )
}