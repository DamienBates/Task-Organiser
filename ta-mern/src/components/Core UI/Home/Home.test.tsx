import Home from "./Home"
import React from "react"
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme'

Enzyme.configure({ adapter: new Adapter() })

describe("home page tests", function () {
    test("text displays", () => {
        expect(Home).toBeDefined();
    });

    test("home page renders without crashing", () => {
        const tree = renderer
            .create(<Home />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})