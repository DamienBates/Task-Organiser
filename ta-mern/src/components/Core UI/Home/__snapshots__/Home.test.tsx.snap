// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page tests home page renders without crashing 1`] = `
<div
  className="MuiBox-root css-0"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "94vh",
      "justifyContent": "center",
      "paddingLeft": "1rem",
      "paddingRight": "1rem",
    }
  }
>
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    Feel free to add, organise, edit and sort tasks to your liking.
  </p>
</div>
`;
